src/AppInstallerCLICore/Workflows/ShowFlow.cpp

```diff
@@ -83,7 +83,8 @@ namespace AppInstaller::CLI::Workflow
         auto releaseNotes = manifest.CurrentLocalization.Get<Manifest::Localization::ReleaseNotes>();
         if (!releaseNotes.empty())
         {
-            info << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelReleaseNotes << ' ' << releaseNotes << std::endl;
+            info << Execution::ManifestInfoEmphasis << Resource::String::ShowLabelReleaseNotes << std::endl;
+            info << "  " << std::regex_replace(releaseNotes, std::regex("\n"), "\n  ") << std::endl;

```

Comment:
1. I like the idea, but if we're doing it maybe we should do the same for any multi-line field? For example, description, agreements, installation notes.
2. I'm a little concerned about how it will look if it has only one line, in the context of everything else being in the same line. Like
```
Label1: Something in one line
Release Notes:
  This is on another line
Label2: In the same line again
```
3. Are we sure this will play nicely in all locales? I'm not an expert in Unicode so I'll trust you if you say it works. I'm thinking something like a multi-byte character where the second byte is the same as `\n` or some other weirdness, but maybe that's not how UTF-8 works.

Reply:
4. I'm pretty sure it wouldn't be officially supported but do we have anybody parsing the output that we could be breaking?

Reply:
I think doing it for all multi-line would be a good idea too; Guess I hadnt noticed it on other fields

I could implement a check to see if it is single line or multi-line, as your representation is accurate that this currently would put single lines onto the next row and indented also

I haven’t tested with other locales, but my understanding is that it explicitly matches the newline character, which is a standard bytecode in UTF-8 and shouldn’t impact multi-byte characters.

Reply:
> 4\. I'm pretty sure it wouldn't be officially supported but do we have anybody parsing the output that we could be breaking?

I think no, because any multi-line release notes would already be breaking output parsing. If anything, this would make it easier to parse release notes using regex

Reply:
@lechacon - I implemented a new function to perform the replacement as well as to return whether a replacement was made. Used that function to determine if a field was single-line or multi-line and handle appropriately for each case, then extended the functionality to description and installation notes